sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -240,23 +243,64 @@ static Read create() {
      *
```

Comment:
You can call any withInstanceId method so you shouldn't say that they need to call the string one.

Please update this here and elsewhere, ditto for withProjectId, ...

Reply:
ok